### DISCUSSION TITLE
Enable Watch for Unraid

### DISCUSSION THREAD

**Discussion by thedinz** [Taxel/PlexTraktSync#834] [Ideas]
I have asked about this but no one knew at the time or chose not to respond, you can please add this to your unraid instructions.

If you want to run Plex Trakt Sync with the watch command and for scrobbling all you need to do is

Edit the container, in the top right, click the toggle at basic view onto advanced view, enter "watch" into the "post arguments" box

**Comment by antonio1475** [comment]
Ah, nice! In Portainer this is in Command and Logging -> Command
I'm adding it to my how-to post for Portainer in HA OS [here](https://github.com/Taxel/PlexTraktSync/discussions/831).

Thank you!!!

**Comment by BenRoc1979** [comment]
Hmmm, it doesnt seem to do anything for me when I put "watch" in the post arguments.

Well, I mean nothing shows in the console except /app #

**Reply by thedinz** [reply]
You still need to run it from the console first, then when rebooting the server or the docker container it should continue to be enabled.

plextraktsync watch

run that from the console

**Comment by glensc** [comment]
@thedinz send the pull request if you mean the instructions in the readme, nobody is going to do that instead of you

**Reply by thedinz** [reply]
I was just trying to help, i have it working fine for me just trying to tell him the possible missing piece.

But if not mistaken that is part of the docs, if you want to scrobble you run traktplexsync watch

If you on unraid, edit the docker app and add "watch" into the "post arguments" field and then run plextraktsync watch and your good to go moving forward, every restart and the scrobbler will continue to be enabled.

**Reply by thedinz** [reply]
I would do the pull request thing your asking me to do, if i knew how lol, sorry.

**Reply by glensc** [reply]
I don't use any of that stuff so I personally don't care. I just pointing you to right direction, to have information available for other users. I'm sure you know how to google "how to make a pull request". and I was responding to the thread author, that is to you.
